File: 은지/백준/1158_요세푸스 문제.py
n, k = map(int, input().split())

answer = "<"
p = [x for x in range(1, n+1)]
i = k-1
while len(p) > 1:        
    v = p.pop(i)
    i = (k+i-1) % len(p)
    answer += str(v) + ", "
v = p.pop(0)
answer += str(v) + ">"

print(answer)

File: 은지/백준/1158_요세푸스 문제.js
const fs = require('fs'); 
const input = fs.readFileSync("/dev/stdin").toString().trim().split(" ").map(v => +v);
n = input[0]
k = input[1]

var answer = "<"
var p = []
for (let x=1; x<=n; x++) {
  p.push(x)
}

var i = k-1
while (p.length > 1) {
  var v = p.splice(i, 1)
  i = (k+i-1) % p.length
  answer += String(v) + ", "
}
v = p.pop()
answer += String(v) + ">"

console.log(answer)